test/printer/index.ts

```diff
@@ -23,7 +31,7 @@ for (const file of files) {
 
     test(`printer: ${file.slice(0, file.length - `.${ending}`.length)}`, async (t) => {
         const actualOutput = await format(input, {
-            parser: (ending === 'html' ? 'svelte' : 'markdown'),
+            parser: ending === 'html' ? 'svelte' : 'markdown',
```

Comment:
```suggestion
            parser: (ending === 'html' ? 'svelte' : 'markdown'),
```

Just so it doesn't track this file.